Update logging page, closes #673

phit opened the pull request:
closes #673

Commit:
Update logging page, closes #673

ST-DDT commented on source/server/spongineer/logs.rst:
```diff
@@ -12,54 +12,48 @@ logfiles from SpongeForge and SpongeVanilla servers including short descriptions
- SpongeForge won't run on Java 6 (``1.6.x``) or Java 7 (``1.7.x``). If you encounter an error stating that you run
- an older Java build than Java 8, please update your JRE to ``1.8.x`` and try again!
+ SpongeForge won't run anything other than Java 8, and only Update 20 or newer. If you encounter an error stating that 
+ you run an older or newer Java build than Java 8, please update your JRE to ``1.8.x`` and try again!
```
> please update your

Maybe use change since it is not necessarily an update.

phit replied:
not happy with these two sentences in general, will see tonight how to rewrite that

if someone has a good idea let me know

ST-DDT reviewed (approved):
I havent checked the log files, but otherwise this looks good to me.

Commit:
rewrite version note

pretty much copied from https://github.com/SpongePowered/SpongeCommon/commit/942fccbea58a6d0822f474b7303e1d26340726af
